### DISCUSSION TITLE
plex universal-watchlist?

### DISCUSSION THREAD

**Discussion by snickers2k** [meisnate12/Plex-Meta-Manager#868] [General]
hey there. 
does anyone know if there is a way to use the plex-universal-watchlist (https://support.plex.tv/articles/universal-watchlist/) as input for sonarr/radarr within PMM? 
thanks

**Comment by meisnate12** [comment]
Idk if the releases have been pushed to master yet but they're working on linking them directly into radarr/sonarr
